packages/cli/config/tsconfig.json

```
@@ -11,7 +11,7 @@
     "incremental": true,
     "isolatedModules": true,
     "jsx": "react",
-    "lib": ["DOM", "DOM.Iterable", "ScriptHost", "ES2020", "ESNext.Promise"],
+    "lib": ["DOM", "DOM.Iterable", "ScriptHost", "ES2020", "ES2020.Promise"],
```

```suggestion
    "lib": ["DOM", "DOM.Iterable", "ScriptHost", "ES2020"],
```

Suggesting that we do this for now